fn_query_tar_network/fn_query_tar_network/util/config.py

```diff
@@ -0,0 +1,18 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_query_tar_network"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_query_tar_network]
+base_url = https://onionoo.torproject.org/details
+flag = Exit  
+data_fields = exit_addresses,or_addresses,host_name
```

Comment:
add a comment to indicate that a comma separated format should be used. Did you test if someone code extra spaces:
exit_address, or_addresses, host_name

don't know if that still works or you have to strip out blanks